libs/RobotKit/tests/RobotController_test_stateCharging.cpp

```diff
@@ -74,6 +74,30 @@ TEST_F(RobotControllerTest, stateChargingDisconnectedEventChargeDidStopGuardIsCh
 	EXPECT_TRUE(rc.state_machine.is(lksm::state::idle));
 }
 
+TEST_F(RobotControllerTest, stateChargingDisconnectedEventBleConnection)
```

Comment:
This case is already covered in `RobotController_test_stateConnected.cpp`, isn't it?

Reply:
Yes but the stateConnected file test does not shows charging entry sequence, which is done here.